README.md

```diff
@@ -155,6 +156,36 @@ svc/zookeeper-client     ClusterIP   10.31.243.173   <none>        2181/TCP
 svc/zookeeper-headless   ClusterIP   None            <none>        2888/TCP,3888/TCP   2m
 ```
 
+### Deploy a sample Zookeeper cluster with Ephemeral storage
+
+Create a Yaml file called `zk.yaml` with the following content to install a 3-node Zookeeper cluster.
+
+```yaml
+apiVersion: "zookeeper.pravega.io/v1beta1"
+kind: "ZookeeperCluster"
+metadata:
+  name: "example"
+spec:
+  replicas: 3        
+  storageType: ephemeral
+```
+
+```
+$ kubectl create -f zk.yaml
+```
+
+After a couple of minutes, all cluster members should become ready.
+
+```
+$ kubectl get zk
+
+NAME      REPLICAS   READY REPLICAS   VERSION   DESIRED VERSION   INTERNAL ENDPOINT    EXTERNAL ENDPOINT   AGE
+example   3          3                 0.2.7     0.2.7             10.100.200.18:2181   N/A                 94s
+```
+>Note: User should only provide either persistence or ephemeral in the spec, if none of the values is specified default is persistence
+
+>Note: We don't guarantee Data Recovery in case of Ephemeral Storage, so the users should keep in mind that in the case of zookeeper pod restarts user might lose the data.
```

Comment:
This is not about data recovery but about cluster recovery. In case of ephemeral storage, the cluster may not be able to come back up if >quorum number of nodes are restarted simultaneously and that is exactly what we should state here.
Did we test if restarts on < quorum number of nodes works with ephemeral storage. In theory it should work, but we should test it and clarify accordingly in the doc.

Reply:
Yes, @pbelgundi you are right I have changed the documentation and I have tested that in case we are restarting less than quorum number of nodes with ephemeral storage we are able to get ensemble.